scripts/tokenize_data.sh

```diff
@@ -0,0 +1,11 @@
+python3 ../tokenize_data.py \
+--tokenized_data_name wikitext-2-v1-tokenized \
+--tokenized_data_folder /home/searlejj/fsl_groups/grp_retnet/compute/data\
```

Comment:
Granted, this should only have to be run once, but searlejj is not jo288. We will have to again decide on whose account and folder in the supercomputer should run the software.